concursus-examples/src/test/java/com/opencredo/concursus/examples/Person.java

```diff
@@ -6,17 +6,16 @@
 import java.time.LocalDate;
 import java.util.Optional;
 import java.util.UUID;
-import java.util.concurrent.CompletableFuture;
 
 @HandlesEventsFor("person")
```

Comment:
Yes, otherwise the proxy generated for the interface won't know what aggregate type to assign to the Events it generates.